keras/layers/recurrent.py

```diff
@@ -66,6 +67,75 @@ def output(self, train):
         return outputs[-1]
 
 
+
+class SimpleBRNN(Layer):
+    '''
+        Fully connected Bi-directional RNN where:
+            Output at time=t is fed back to input for time=t+1 in a forward pass
+            Output at time=t is fed back to input for time=t-1 in a backward pass
+    '''
+    def __init__(self, input_dim, output_dim,
+        init='uniform', inner_init='orthogonal', activation='sigmoid', weights=None,
+        truncate_gradient=-1,  return_sequences=False):
+        self.init = initializations.get(init)
+        self.inner_init = initializations.get(inner_init)
+        self.input_dim = input_dim
+        self.input_dim = input_dim
```

Comment:
Could you elaborate a bit on this? I see the two duplicate lines as redundant (not sure how they entered the picture), but I'm not sure how much you wan to inherit, especially given SimpleRNN is a demo implementation.